config.py.example

```diff
@@ -9,6 +9,7 @@ LANGUAGE = 'EN'  # ISO 639-1 codes EN, DE, FR, and ZH currently supported.
 MAP_START = (12.3456, 34.5678)
 MAP_END = (13.4567, 35.6789)
 GRID = (2, 2)  # row, column
+# DISABLE_MARKERS = [0, 1, 2]
```

Comment:
For better sounding, please name it `DISABLED_WORKERS`. It can also be initialized as an empty list (without # indicating comment).

Please also add it to `REQUIRED_SETTINGS` in both worker.py and web.py (at the top).